app/api/privy/integrations/api.rb

```diff
@@ -0,0 +1,7 @@
+module Privy
+  module Integrations
+    class API < Grape::API
+      mount Discord::API
+    end
```

Comment:
Perhaps we should also have a prefix for this too?

```suggestion
    class API < Grape::API
      prefix '/integrations'
      mount Discord::API
    end
```

Reply:
It doesn't work to build the /api/integrations/discord/interactions route this way (but I think we don't really like it):

```ruby
module Privy
  class API < Grape::API
    format :json
    formatter :json, Grape::Formatter::Jbuilder
    mount Messages::API
    mount Integrations::API => '/integrations'
    mount Ping
  end
end
```

Reply:
cool, let's go that way. It's better than having to always remember and repeat the `/integrations` in all child routes

Reply:
Done!